### ISSUE TITLE
The input caret always remains at initial position on Samsung devices

### ISSUE THREAD

**Issue by valchev** [telerik/kendo-ui-core#106]
The input caret always remains at initial position on Samsung devices.
The issue can be reproduced in the "Basic usage" demo page with a Samsung device and mobile Google Chrome.

The issue is specific to the Samsung devices (cannot be reproduced on Google Nexus/Motorola device).

**Comment by ppreget** [comment]
This bug is affecting a lot of our customer base; why has this not been addressed yet and even been removed from 2015.Q1? Can you please update us as soon as possible.

**Comment by KirilNN** [comment]
Hi, 

After detailed investigation, I am afraid that we have not discovered any reliable means to address the problem. As it seems to be caused by a vendor specific browser modification, that currently cannot be worked around.

**Comment by architech99** [comment]
Regarding this, I ran into the same issue and for a tablet/kiosk product we're working on for a client. During our beta testing, we found it behaved inconsistently across a few different devices (worked fine on an Azpen 10 inch tablet, but pushed numbers to the right on a Samsung Tab 4 and bounced all over on a Nexus 5X). After finding this ticket marked closed and won't fix, I started looking for alternatives.

I found this plugin for jQuery which I was able to implement and use to get around the issue with the Masked Text Box. 

http://digitalbush.com/projects/masked-input-plugin/
